tests/all/func.rs

```diff
@@ -1422,3 +1422,178 @@ fn calls_with_funcref_and_externref() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test]
+#[cfg_attr(miri, ignore)]
+fn typed_v128() -> anyhow::Result<()> {
```

Comment:
Shouldn't this be `cfg`ed the same way as the `WasmTy` implementation for `V128` is?